src/generators/css-filter.ts

```diff
@@ -5,12 +5,22 @@ import {parseArray, formatArray} from '../utils/text';
 import {compareURLPatterns, isURLInList} from '../utils/url';
 import {createTextStyle} from './text-style';
 import {FilterConfig, InversionFix} from '../definitions';
+import { compareChromeVersions, getChromeVersion, isChrome } from '../utils/platform';
 
 export enum FilterMode {
     light = 0,
     dark = 1
 }
 
+export function hasRootIssue() {
+    const chromeVersion = getChromeVersion();
+    return Boolean(
+        chromeVersion &&
+        isChrome() &&
+        compareChromeVersions(chromeVersion, "81.0.4035.0") >= 0
+    )
```

Comment:
I guess that `npm run code-style` didn't work for some reason. Maybe file patterns didn't work on Windows because of `/` in paths, or maybe single quotes should be replaced with `\"`.

Reply:
I will look into that

Reply:
![](https://i.imgur.com/vN6CBX3.png)